apps/cli/src/commands/run.ts

```diff
@@ -20,6 +20,10 @@ export default class Run extends Command {
             description: "duration of an epoch (in seconds)",
             default: 86400,
         }),
+        "no-backend": Flags.boolean({
+            description: "disable backend and snapshot check",
```

Comment:
Set the `summary` field to `run a node without the application code.`
In the `description`, which is a long description use:

```
Run a node without the application code. Application must be executed by the developer on the host machine, fetching inputs from the node running at http://localhost:5004
```

Or whatever port that will be.